openapi-v1.yaml

```diff
@@ -2628,6 +2628,71 @@ definitions:
       - array:read-write
       - array:admin
 
+  GroupAsset:
+    description: an asset of a group. Can be array notebook ml_model or udf
+    type: object
+    properties:
+      name:
+        description: the name of the asset
+        type: string
+      namespace:
+        description: the namespace of the asset
+        type: string
+
+  GroupInfo:
+    description: A group of assets
+    type: object
+    properties:
+      path:
```

Comment:
Do you mean uri here? Or is this some physical path?

Reply:
This is the path of the group. `{namespace}/a/b`